packages/cli/src/commands/runAndroid/__fixtures__/sampleBuild.gradle

```diff
@@ -0,0 +1,150 @@
+apply plugin: "com.android.application"
+
+import com.android.build.OutputFile
+
+/**
+ * The react.gradle file registers a task for each build variant (e.g. bundleDebugJsAndAssets
```

Comment:
what do you think about removing irrelevant details of fixtures like comments or some setup code?

Reply:
I left them in to make sure the test had something close to a real file to parse. I can strip it down more if you'd like.

Reply:
I think it's maybe better this way, because we can copy-pase from RN template.
Btw, isn't there a way to get the applicationId from `adb` instead of parsing the manifest?

Reply:
`adb` would only be able to enumerate installed packages on the target device.  

We could use `aapt` to dump the applicationId from the apk, but that would still require parsing. That would also introduce dependency on `aapt` being available from the command line, which by default it is not. I think parsing the manifest is probably the best compromise.